.golangci.yml

```diff
@@ -0,0 +1,14 @@
+run:
+  tests: false
+linters: 
+  disable-all: true
+  enable:
+    - misspell
+    - gofmt
+    - goimports
+    - golint
+    - ineffassign
+    - deadcode
+    - unconvert
+    - govet
+      
```

Comment:
@dperny 
Do you want to exclude auto generated protobuf files `pb.go` and vendor directory?
Similar to `gometalinter` , `golangci` supports a deadline flag

Reply:
We probably do, but currently those actually pass linting, so I didn't worry about actually doing it.

Reply:
I wonder how much extra time do they add ? if it's significant , shall we add them now?